concepts/complex-numbers/about.md

```diff
@@ -1,2 +1,182 @@
+(-0.5+0.5j)
+
+>>> z1 * z2  # multiplication
+6.25j
+
+>>> z1 / z2  # division
+(0.96+0.28j)
+
+>>> z1 ** 2  # exponentiation
+(-1.75+6j)
+
+>>> 2 ** z1  # another exponentiation
+(0.5188946835878313+2.7804223253571183j)
+
+>>> 1j ** 2 # j is the square root of -1
+(-1+0j)
+```
+
+Explaining the rules for complex multiplication and division is out of scope here.
+Any introduction to complex numbers will cover this.
+Alternatively, Exercism has a `Complex Numbers` practice exercise where you can implement this from first principles.
+
+Integer division is ___not___ possible on complex numbers, so the `//` and `%` operators and `divmod()` function will fail.
+
+There are two functions that are useful with complex numbers:
+- `conjugate()` simply flips the sign of the complex part.
+Because of the way complex multiplication works, this is more useful than you might think.
+- `abs()` is guaranteed to return a real number with no imaginary part.
+
+```python
+>>> z1
+(1.5+2j)
+
+>>> z1.conjugate() # flip the z1.imag sign
+(1.5-2j)
+
+>>> abs(z1) # sqrt(z1.real ** 2 + z1.imag ** 2)
+2.5
+```
+
+## The `cmath` module
+
+The Python standard library has a `math` module full of useful functionality for working with real numbers.
+It also has an equivalent `cmath` module for complex numbers.
+
+Details are available in the [`cmath`][cmath] module documents, but the main categories are:
+- conversion between Cartesian and polar coordinates
+- exponential and log functions
+- trig functions
+- hyperbolic functions
+- classification functions
+- useful constants
+
+```python
+>>> import cmath
+
+>>> euler = cmath.exp(1j * cmath.pi) # Euler's equation
+
+>>> euler.real
+-1.0
+>>> round(euler.imag, 15) # round to 15 decimal places
+0.0
+```
+
+So a simple expression with three of the most important constants in nature `e`, `i` (or `j`) and `pi` gives the result `-1`. 
+Some people believe this is the most beautiful result in all of mathematics.
+It dates back to around 1740.
+
+-----
+
+## Optional section: a Complex Numbers FAQ
+
+This part can be skipped, unless you are interested.
+
+### Isn't this some strange new piece of pure mathematics?
+
+It was strange and new in the 16th century.
+
+500 years later, it is central to most of engineering and the physical sciences.
+
+### Why would anyone use these?
+
+It turns out that complex numbers are the simplest way to describe anything that rotates or anything with a wave-like property.
+
+You can see things rotate.
+Complex numbers may not make the world go round, but they are great for explaining what happens as a result of the world going round: look at any satellite image of a major storm.
+
+Less obviously, sound is wave-like, light is wave-like, radio signals are wave-like, The economy of your home country is at least partly wave-like.
+
+A lot of this can be done with trig functions (`sin()` and `cos()`) but that gets messy quickly.
+Complex exponentials are ___much___ easier to work with.

```

Comment:
```suggestion
Complex exponentials are ___much___ easier to work with, and moving these waves to a representation on the complex number plane simplifies calculations.

```

Reply:
We didn't mention the complex plane before, and I had already wondered if we should. I resisted the temptation to include a graphic showing the complex plane and unit circle, and magnitude vs phase. Maybe a Wikipedia link?

Too much of this and we'll be discussing the Cauchy-Riemann equations and having to explain partial differential equations...

Reply:
I'm still not sure what to do about this.

Reply:
Me either.  All of the examples I've found have shown the unit circle and how things translate (rotate) when put on the complex plane.  [this rundown](https://realpython.com/python-complex-numbers/) from Real Python uses a (contrived) example of the Bermuda Triangle.  Very tempted to adapt it with a different region of the globe.  Or maybe we use a simple geometric form and how it can be flipped and rotated easier on the complex plane.  URG.  More thought needed on my part, clearly.